### PR TITLE
[bitnami/kafka] Allow loadBalancerClass to be customized for the kafka chart

### PR DESCRIPTION
**Pull request by jamezrin**
### Description of the change
This PR allows the `loadBalancerClass` to be customized via the chart values in the `kafka` chart.

### Benefits

Allows the following values to be set, setting the `loadBalancerClass` to be set in the respective `Service`
- `service.loadBalancerClass`
- `externalAccess.controller.service.loadBalancerClass`
- `externalAccess.broker.service.loadBalancerClass`

### Possible drawbacks

None

### Applicable issues

None

### Additional information

N/A

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [X] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by juan131 — APPROVED** [review]
Thanks for the PR! LGTM

File: bitnami/kafka/templates/broker/svc-external-access.yaml [hunk 1]
```diff
@@ -31,6 +31,9 @@ metadata:
 spec:
   type: {{ $.Values.externalAccess.broker.service.type }}
   {{- if eq $.Values.externalAccess.broker.service.type "LoadBalancer" }}
+  {{- if (not (empty .Values.externalAccess.broker.service.loadBalancerClass)) }}
+  loadBalancerClass: {{ .Values.externalAccess.broker.service.loadBalancerClass }}
```
Comment (juan131) [inline bitnami/kafka/templates/broker/svc-external-access.yaml:35]: A small change is required here:
```suggestion
  {{- if (not (empty $.Values.externalAccess.broker.service.loadBalancerClass)) }}
  loadBalancerClass: {{ $.Values.externalAccess.broker.service.loadBalancerClass }}
```

**Review by juan131 — CHANGES_REQUESTED** [review]

**Review by juan131 — DISMISSED** [review]
LGTM

**Comment by jamezrin** [comment]
Sorry for the multiple changes, I had to also apply that fix for `controller-eligible` and reorder, it's all good now.

**Review by juan131 — APPROVED** [review]

**Pull request merged**